.github/workflows/nodejs.yml

```diff
@@ -6,8 +6,10 @@ name: Build & Test
 on:
   push:
     branches: [ main ]
+    paths-ignore: ["ports/**"]
```

Comment:
This should prevent running NodeJS CI checks on ports changes, but then ports must not depend on NodeJS files, so be careful.